ansibleinviewer/inventoryparser.py

```diff
@@ -68,4 +105,11 @@ def parse_inventory_data_dict(self, inventory_data: dict) -> Dict:
         return hosts_dict
 
     def get_hosts(self) -> List[Host]:
-        return list(self.hosts)
+        if self.target_hosts:
+            host_list = []
+            for host in self.hosts:
+                if host.hostname in self.target_hosts:
+                    host_list.append(host)
+            return host_list
+        else:
+            return list(self.hosts)
```

Comment:
I think we should pass the group as an argument to the get_hosts method instead of the __init__.

The groups slicing is cool, but I do not think it will be that useful for the user. He will probably not know which hosts are the first 4 hosts, or the last 3 in the inventory. I was thinking about selecting the whole group as one way of selecting hosts and providing a list of hostnames to show as the other.

Reply:
Done